iotics/lib/identity/validation/identity.py

```diff
@@ -20,6 +23,17 @@ def validate_identifier(did: str):
         if result is None:
             raise IdentityValidationError(f'Identifier does not match pattern {did} - {IDENTIFIER_ID_PATTERN}')
 
+        did_bytes = base58.b58decode(did[len(IDENTIFIER_PREFIX):], alphabet=base58.BITCOIN_ALPHABET)
```

Comment:
following up the comment above, should this magic number be put in a constant (since it's repeated in many places or maybe have a `id_sdk.did_decode`/`id_sdk.did_encode`  functions that wrap `base58.b58decode/b58encode(alphabet=base58.BITCOIN_APLHABET)` ?
just saying in order to have our custom logic somewhere in one place only